fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -152,17 +152,36 @@ def self.buildlog_path
     #  running system
     def self.xcode_path
       return "" unless self.is_mac?
-      `xcode-select -p`.delete("\n") + "/"
+
+      if self.xcode_server?
+        # Xcode server always creates a link here
+        return "/Library/Developer/XcodeServer/CurrentXcodeSymlink/Contents/Developer"
+      end
+
+      return `xcode-select -p`.delete("\n") + "/"
+    end
+
+    def self.xcode_server?
+      # XCS is set by Xcode Server
+      is_xcode_server = ENV["XCS"] || 0
```

Comment:
This whole method could also be `ENV["XCS"].to_i == 1`, right?

Reply:
yup